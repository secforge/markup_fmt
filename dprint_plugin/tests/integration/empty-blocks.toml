# Test empty and self-closing custom blocks
[vue_custom_block]
default = "lang-attribute"
docs = "none"
metadata = "squash"
