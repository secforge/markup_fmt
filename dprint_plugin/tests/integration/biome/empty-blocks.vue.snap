---
source: dprint_plugin/tests/integration.rs
assertion_line: 218
---
<template>
  <div>Test empty and self-closing blocks</div>
</template>

<!-- Self-closing block with lang-attribute mode -->
<i18n lang="json" />

<!-- Empty block with lang-attribute mode -->
<i18n lang="json"></i18n>

<!-- Self-closing block with none mode -->
<docs />

<!-- Empty block with none mode -->
<docs></docs>

<!-- Self-closing block with squash mode -->
<metadata />

<!-- Empty block with squash mode -->
<metadata></metadata>
